internal/usagestats/ide_extensions.go

```diff
@@ -0,0 +1,167 @@
+package usagestats
+
+import (
+	"context"
+
+	"github.com/sourcegraph/sourcegraph/internal/database"
+	"github.com/sourcegraph/sourcegraph/internal/types"
+)
+
+func GetIdeExtensionsUsageStatistics(ctx context.Context, db database.DB) (*types.IdeExtensionsUsage, error) {
+	stats := types.IdeExtensionsUsage{}
+
+	usageStatisticsByIdext := []*types.IdeExtensionsUsageStatistics{}
+
+	rows, err := db.QueryContext(ctx, ideExtensionsPeriodUsageQuery, timeNow())
+	if err != nil {
+		return nil, err
+	}
+
+	defer rows.Close()
+
+	for rows.Next() {
+		ideaExtensionUsage := types.IdeExtensionsUsageStatistics{}
```

Comment:
```suggestion
		ideExtensionUsage := types.IdeExtensionsUsageStatistics{}
```

Unless this refers to Intellij IDEA 😅